lib/tasks/single_run.rake

```diff
@@ -65,4 +65,17 @@ namespace :single_run do
       end
     end
   end
+
+  desc '2017-08-15: Change project_name to project_id'
+  task add_project_id_to_team: :environment do
+    Team.all.each do |team|
+      next team unless team.project_name.present?
+      begin
+        team.project = Project.where(season_id: team.season_id).find_by!(name: team.project_name)
```

Comment:
Actually, I'd rather not. It would require to downcase both the `team.project_name` and each `project.name`.
I'd say the risk of this particular misspelling is very low, compared to all possible misspellings. Either way, problems will pop up in the console and will not go unnoticed.
Is it okay with you if I don't downcase?